frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -58,11 +60,14 @@ const EmptyStateLoader: React.FC<EmptyStateLoaderProps> = ({ resources, loaded,
   return (
     <ODCEmptyState
       title="Add"
-      {...noWorkloads && {
-        hintBlockTitle: 'No workloads found',
-        hintBlockDescription:
-          'To add content to your project, create an application, component or service using one of these options.',
-      }}
+      hintBlock={
+        noWorkloads && (
```

Comment:
Oh I missed the conditional - this will return `false` through the `hintBlock` prop and will remove the default text - which I don't think is desired.

```
false && <HintBlock ... />
// false (needs to be undefined in order to use a defaultProp value)
```

Suggest just handling it this way (but feel free to handle it another way if you want):
```
  if (noWorkloads) {
    return (
      <ODCEmptyState
        title="Add"
        hintBlock={
          <HintBlock
            title="No workloads found"
            body="To add content to your project, create an application, component or service using one of these options."
          />
        }
      />
    );
  }

  return <ODCEmptyState title="Add" />;
```